wheel_matrix.py

```diff
@@ -19,6 +19,7 @@ class Package:
 def main() -> None:
     freebsd_pythons = {
         '13.2': ['3.9', '3.11'],
+        '14.0': ['3.11'],
```

Comment:
Wheels are needed for Python 3.9 also:

```suggestion
        '14.0': ['3.9', '3.11'],
```

Reply:
For anyone wondering why, the reason is that we install packages in virtual environments too. So it's not enough for these to be available as OS packages for a given Python.

Reply:
i was looking to avoid all wheels for all versions, since 3.9 does not need cryptography

Reply:
Python 3.9 does need cryptography (see my previous comment).

Reply:
I've unresolved it, since Python 3.9 is needed.

Reply:
not sure what you mean, it is there, do you want me to remove 3.9?

Reply:
nvmd, i was not pushign to the PR branch